truffle/test/helpers.js

```diff
@@ -0,0 +1,5 @@
+const BN = require("bn.js");
+
+module.exports.fundsToInt = function(funds) {
+    return funds.div(new BN("1000000000000000000")).toNumber();
```

Comment:
Why not use BigInt and drop the dependency on bn altogether?

Reply:
What comes from the blockchain network is a BN struct, i cant divide a BN with a BigInt